cmd/subnetcmd/configure.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package subnetcmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/spf13/cobra"
+)
+
+// avalanche subnet configure
+func newConfigureCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "configure [subnetName]",
+		Short:        "",
+		Long:         ``,
+		SilenceUsage: true,
+		RunE:         configure,
+		Args:         cobra.ExactArgs(1),
+	}
+	return cmd
+}
+
+func configure(cmd *cobra.Command, args []string) error {
+	chains, err := validateSubnetNameAndGetChains(args)
+	if err != nil {
+		return err
+	}
+	subnetName := chains[0]
+
+	const (
+		subnetConf = "Subnet config"
+		chainConf  = "Chain config"
+		abort      = "Abort"
+	)
+
+	ux.Logger.PrintToUser("The " + logging.Cyan.Wrap("subnet") + logging.Reset.Wrap(" config file applies to *all* VMs in a subnet"))
```

Comment:
I'm not sure we want these messages right now since we only support one chain per subnet. It might just confuse people

Reply:
But at the same time we are asking the user to select exactly those things - either subnet or chain config. If she is confused by the explication messages, then she shouldn't be using this command at all in my opinion. I vote to keep.

Reply:
Do they share the same settings? I thought they were more or less totally different?

Reply:
I still think we should comment this out. I think it just adds confusion.